pytorch_lightning/plugins/environments/lsf_environment.py

```diff
@@ -56,13 +56,15 @@ def is_using_lsf() -> bool:
     def creates_processes_externally(self) -> bool:
         return True
 
-    def master_address(self):
-        """The master address is read from a list of hosts contained in the environment variable `LSB_HOSTS`."""
-        return self._master_address
+    @property
+    def main_address(self):
```

Comment:
nit: add return typing, same for other modified occurrences in this file

Reply:
ah good call, just realized we don't have mypy for cluster envs. should be easy to do.

Reply:
added types but need to send PR to properly make mypy pass on some methods